pkg/collector/scaling_schedule_collector_test.go

```diff
@@ -80,7 +86,45 @@ func TestScalingScheduleCollector(t *testing.T) {
 			expectedValue: 100,
 		},
 		{
-			msg: "Return the default value (0) for one time config - 30 seconds after",
+			msg: "Return the scaled value (50) for one time config - 20 seconds before starting",
```

Comment:
nit: the comment doesn't align with the expected value: 50/67

Reply:
Oops, will fix. Thanks!